examples/jacobi.cpp

```diff
@@ -456,7 +456,7 @@ int main(int RAJA_UNUSED_ARG(argc), char **RAJA_UNUSED_ARG(argv[]))
     iteration++;
   }
   hipDeviceSynchronize();
-  hipErrchk(hipMemcpy( I, d_I, NN * sizeof(double), hipMemcpyDeviceToHost ));
+  CAMP_HIP_API_INVOKE_AND_CHECK(hipMemcpy, I, d_I, NN * sizeof(double), hipMemcpyDeviceToHost);
```

Comment:
We are using the new macro in the examples now, should we do this or handles errors in another way as it does look a bit strange.

Reply:
Also if we're going to use it we should be consistent and use it on every hip/cuda call.